html/mycakeapp/webroot/css/movie.css

```
@@ -0,0 +1,190 @@
+/* color */
+.back-orange {
+    background-color: #f5ae0b;
+}
+.back-gray {
+    background-color: #00000014;
+}
+header,
+footer {
+    background-color: #262626;
+}
+.title,
+div[class^="frame"],
+.back-gray {
+    color: #707070;
+}
+header a,
+footer a,
+.back-orange {
+    color: #ffffff;
+}
+
+/* button */
+button {
+    display: block;
+    font-size: 16px;
+    box-shadow: 0px 2px 2px #00000015;
+    border: none;
+    height: 48px;
+    letter-spacing: 0.44px;
+}
+/* 幅いっぱいのボタン */
+.normal-button {
+    margin: 0 auto;
+    width: 336px;
+    font-weight: bold;
+}
+/* 一番小さいボタン */
+.mini-button {
+    height: auto;
+    padding: 7px 22px;
+    font-weight: bold;
+}
+/* どっちも半分 */
+.half-button button:first-of-type {
+    margin-right: 30px;
+}
+/* 1:2のボタン */
+.division-button .back-gray {
+    width: 128px;
+}
+.division-button button:first-of-type {
+    margin-right: 16px;
+}
+/* halfとdivision共通 */
+/* 背景指定した2つのボタンを.flex.division(.halfbutton)のdivで囲う */
+.division-button,
+.half-button {
+    justify-content: center;
+}
+.half-button button,
+.division-button .back-orange {
+    width: 192px;
+}
+
+/* 共通 */
+* {
+    padding: 0;
+    margin: 0;
+}
+body * {
+    font-family: "Noto Sans JP", sans-serif;
+}
+.flex {
+    display: flex;
+}
+body {
+    min-height: 1080px;
+    min-width: 1920px;
```

これだとwidthが1920px以下のブラウザ(ex.macbook)で開いた場合、横にスクロールしなければいけなくなります。
```suggestion
    max-width: 1920px;
```
が最適かと思われます。
また、それによってheader,footerの横の余白が無くなってしまうので、そこの修正も必要となりますのでご注意を

---

ご指摘ありがとうございます！！修正いたします！